app/features/dynamo/doc_type_processors/test_ppt_processor.py

```diff
@@ -0,0 +1,35 @@
+# test_ppt_processor.py
+import pandas as pd
+from ppt_processor import PPTProcessor
+from langchain_google_vertexai import VertexAI  # Make sure this import is correct or mock it
+
+# Mock the VertexAI object if not available
+class MockVertexAI:
```

Comment:
Why are we mocking the VertexAI API if its not available with an empty class? Instead do robust error handling and edge case handling.